rotkehlchen/tests/api/test_misc.py

```diff
@@ -260,3 +265,17 @@ def test_manage_ethereum_nodes(rotkehlchen_api_server):
         },
     )
     assert_proper_response_with_result(response)
+
+
+def test_configuration(rotkehlchen_api_server):
+    """Test that the configuration endpoint returns the expected information"""
+    response = requests.get(api_url_for(rotkehlchen_api_server, 'configurationsresource'))
+    response_json = assert_proper_response_with_result(response)
+    assert response_json['max_size_in_mb_all_logs']['value'] == DEFAULT_MAX_LOG_SIZE_IN_MB
+    assert response_json['max_size_in_mb_all_logs']['is_default'] is True
+    assert response_json['max_logfiles_num']['is_default'] is True
+    assert response_json['max_logfiles_num']['value'] == DEFAULT_MAX_LOG_BACKUP_FILES
+    assert response_json['sqlite_instructions']['is_default'] is True
+    assert response_json['sqlite_instructions']['value'] == DEFAULT_SQL_VM_INSTRUCTIONS_CB
+    assert response_json['sleep_secs']['is_default'] is False
+    assert response_json['sleep_secs']['value'] == 60
```

Comment:
Where did you set this value?

Reply:
https://github.com/rotki/rotki/blob/develop/rotkehlchen/tests/utils/args.py#L21

Reply:
Wooo this is not good. It was done like that only cause the default args thing only touched things used in tests, and this was not.

This will break the first time we fix the default_args function.

Perhaps add a note in the line you quoted, that you do this so in the tests you have a non-default argument? Again this can be a nitpick for next PR or you can do both this and the other thing now.

Reply:
you did not do exactly what I asked. By doing what you do right now more changes are needed which I will push.